src/adafruit_blinka/microcontroller/tegra/pin.py

```diff
@@ -0,0 +1,132 @@
+import sys
+import Jetson.GPIO as GPIO
+sys.path.append("/opt/nvidia/jetson-gpio/lib/python")
```

Comment:
this is a little unusual - why is it here? you've already imported the GPIO library?